### DISCUSSION TITLE
[@types/ws] WebSocket pause, resume and isPaused missing

### DISCUSSION THREAD

**Discussion by Barnabas** [DefinitelyTyped/DefinitelyTyped#58999] [Issues with a @types package]
Hi, I noticed that in [release 8.3.0](https://github.com/websockets/ws/releases/tag/8.3.0) the [pause](https://github.com/websockets/ws/blob/master/doc/ws.md#websocketpause) and [resume](https://github.com/websockets/ws/blob/master/doc/ws.md#websocketresume) methods and the [isPaused](https://github.com/websockets/ws/blob/master/doc/ws.md#websocketispaused) property were added. These are not yet in the type definition as of 8.5.0, though. I thought you'd like to know.

[a comment by an automated account is not shown]

**Comment by k-yle** [comment]
fixed in #59028
